x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/130_spatial.yml

```diff
@@ -134,7 +150,15 @@ geo_point unsortable with limit from row:
 ---
 values unsupported for geo_point:
   - do:
-      catch: '/.+argument of \[VALUES\(location\)\] must be \[boolean, date, double, integer, ip, keyword, long, null, text, unsigned_long or version\].+/'
+      catch: '/.+argument of \[VALUES\(location\)\] must be .+/'
```

Comment:
The version of this in main fails when adding new data types, as that list is auto-generated out of non-spatial types. Unfortunately, explicitly adding types to this list would then require gating this test on each data type in the error message.  Making this less specific means it can run across more sets of types.